wgengine/userspace.go

```diff
@@ -606,7 +607,7 @@ func (e *userspaceEngine) maybeReconfigWireguardLocked(discoChanged map[key.Publ
 	// based on the full config. Prune off all the peers
 	// and only add the active ones back.
 	min := full
-	min.Peers = nil
+	min.Peers = make([]wgcfg.Peer, 0, e.lastNMinPeers)
```

Comment:
this number might be `len(e.lastCfgFull.Peers)-len(e.trimmedDisco)`? maybe I'm overthinking. :)

Reply:
Maybe, but this logic is tricky. I'd rather keep it simple.